user/applications/product_id_and_version/application.cpp

```diff
@@ -5,13 +5,19 @@
 //
 PRODUCT_VERSION(3);
 
+#if PLATFORM_ID == PLATFORM_ESOMX
+	hal_pin_t toggle_pin = LED_PIN_USER;
```

Comment:
This should be defined for all platforms and this is an internal app. Can use `LED_PIN_USER` universally.

Reply:
Ok, I didnt know if this define was available on gen2 platforms when we backport this PR to 4.x

Reply:
Well, good news is that there is no Gen 2 support in 4.x :)

Reply:
oh, right 🙃